File: features/transaction/buyerorder/src/main/java/com/tokopedia/buyerorder/detail/data/AdditionalInfo.java
package com.tokopedia.buyerorder.detail.data;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by baghira on 11/05/18.
 */

public class AdditionalInfo {
    @SerializedName("label")
    @Expose
    private String label;
    @SerializedName("value")
    @Expose
    private String value;

    public String label() {
        return label;
    }

    public String value() {
        return value;
    }

    @Override
    public String toString() {
        return "[AdditionalInfo:{"
                + "label="+label +" "
                + "value="+value +" "
                + "}]";
    }
}

File: features/transaction/checkout/src/main/java/com/tokopedia/checkout/data/model/response/shipment_address_form/promo_checkout/AdditionalInfo.kt
package com.tokopedia.checkout.data.model.response.shipment_address_form.promo_checkout

import javax.annotation.Generated
import com.google.gson.annotations.SerializedName

@Generated("com.robohorse.robopojogenerator")
data class AdditionalInfo(

		@field:SerializedName("sp_ids")
	val spIds: List<Int?>? = null,

		@field:SerializedName("message_info")
	val messageInfo: MessageInfo? = null,

		@field:SerializedName("error_detail")
	val errorDetail: ErrorDetail? = null,

		@field:SerializedName("empty_cart_info")
	val cartEmptyInfo: CartEmptyInfo? = null,

		@field:SerializedName("usage_summaries")
		val listUsageSummaries: List<UsageSummaries>? = null
)